### ISSUE TITLE
[cppwinrt] Provide MSBuild integration at parity with NuGet package

### ISSUE THREAD

**Issue by davidmatson** [microsoft/vcpkg#31421]
### Is your feature request related to a problem? Please describe.

Currently, cppwinrt via vcpkg isn't as full-featured on MSBuild as cppwinrt via NuGet; installing this package isn't enough to get it to run the tool on WinMD files, for example.

### Proposed solution

Include the MSBuild props/targets from upstream for MSBuild consumers, so they run in the same way for projects with MSBuild integration enabled.

(Assuming vcpkg has a way to include MSBuild targets in projects that integrate with vcpkg via MSBuild.)

**Comment by jimwang118** [comment]
Are you saying that `cppwinrt` installed via `vcpkg` cannot handle `WinMD` files?

**Comment by davidmatson** [comment]
No, I'm saying that, for MSBuild projects (.vcxproj), if you use cppwinrt via the NuGet package and reference a WinMD file, it just works and automatically produces the correct cppwinrt headers from that WinMD file. If you use cppwinrt via vcpkg, that doesn't happen.
